applications/fishing-map/src/features/map/map.selectors.ts

```diff
@@ -146,6 +149,20 @@ export const getWorkspaceGeneratorsConfig = createSelector(
           if (dataset?.source) {
             generator.attribution = dataset.source
           }
+          if (dataview.config?.type === Generators.Type.UserContext) {
+            if (
+              dataset.configuration?.propertyToInclude &&
```

Comment:
This condition is in other places too, what is it used for? could we add a method to the interface that defines a contract for this condition and then implement it when the dataset configuration is created, makes sense?

Reply:
The same for the workaround before, as it won't be needed when we have the different layer types feature it will be removed

Reply:
Next time I'll leave some comments on the code.